demos/dev_mode_key_provisioning/include/aws_dev_mode_key_provisioning.h

```diff
@@ -23,87 +23,192 @@
  * http://www.FreeRTOS.org
  */
 
+
+
 /**
  * @file  aws_dev_mode_key_provsioning.h
  * @brief Provisioning example code for developers.
- * 
- * Helpers for importing device private key and device 
+ *
+ * Helpers for importing device private key and device
  * certificate for use with AWS connectivity libraries.
- * 
+ *
  * \warn This code is provided for example purposes only, and
  * should not be used in production code.
  */
 
+
 #ifndef _AWS_DEV_MODE_KEY_PROVISIONING_H_
 #define _AWS_DEV_MODE_KEY_PROVISIONING_H_
-
 #include "iot_pkcs11.h"
 
 typedef struct ProvisioningParams_t
 {
-    uint32_t ulClientPrivateKeyType;       /**< The type of key to be provisioned, CKK_RSA or CKK_EC */
-    uint8_t * pcClientPrivateKey;          /**< Pointer to the device private key in PEM format.  
-                                                See tools/certificate_configuration/PEMfileToCString.html 
-                                                for help with formatting.*/
-    uint32_t ulClientPrivateKeyLength;     /**< Length of the private key data, in bytes. */
-    uint8_t * pcClientCertificate;         /**< Pointer to the device certificate in PEM format.
-                                                See tools/certificate_configuration/PEMfileToCString.html 
-                                                for help with formatting.*/ 
-    uint32_t ulClientCertificateLength;    /**< Length of the certificate in bytes. */
+    uint8_t * pucClientPrivateKey;      /**< Pointer to the device private key in PEM format.
+                                         *   See tools/certificate_configuration/PEMfileToCString.html
+                                         *   for help with formatting.*/
+    uint32_t ulClientPrivateKeyLength;  /**< Length of the private key data, in bytes. */
+    uint8_t * pucClientCertificate;     /**< Pointer to the device certificate in PEM format.
+                                         *   See tools/certificate_configuration/PEMfileToCString.html
+                                         *   for help with formatting.*/
+    uint32_t ulClientCertificateLength; /**< Length of the device certificate in bytes. */
+    uint8_t * pucJITPCertificate;       /**< Pointer to the JITP certificate in PEM format.  If no
```

Comment:
Can you please expand this acronym in the comment? It would be nice to also link to the IoT Core feature in question, since that's important for the customer transition to production.